src/DotNetStatus/Controllers/DataController.cs

```diff
@@ -0,0 +1,224 @@
+        }
+
+        [Route("BreakingChanges/{orderBy?}")]
+        public async Task<IActionResult> GetBreakingChangesAsync(BreakingChangeSortFieldEnum orderBy = BreakingChangeSortFieldEnum.Id)
+        {
+            string query = "/Api/data/BreakingChanges";
+            var data = await _jsonData.GetJsonDataAsync<IEnumerable<BreakingChange>>(query);
+
+            return PartialView("BreakingChanges", SortBreakingChanges(data, orderBy));
+        }
+
+        [Route("ApiPortSummary/{orderBy?}")]
+        public async Task<IActionResult> GetApiPortSummaryAsync(KeyValueSortFieldsEnum orderBy = KeyValueSortFieldsEnum.Key)
+        {
+            return PartialView("KeyValuePair", await GetApiPortSummaryJsonAsync(orderBy));
+        }
+
+        [Route("AnalyzersSummary/{orderBy?}")]
+        public async Task<IActionResult> GetAnalyzersSummaryAsync(KeyValueSortFieldsEnum orderBy = KeyValueSortFieldsEnum.Key)
+        {
+            return PartialView("KeyValuePair", await GetAnalyzersSummaryJsonAsync(orderBy));
+        }
+
+        [Route("NotCoveredByAny/{orderBy?}")]
+        public async Task<IActionResult> GetNotCoveredByAnyAsync(BreakingChangeSortFieldEnum orderBy = BreakingChangeSortFieldEnum.Id)
+        {
+            return PartialView("BreakingChanges", SortBreakingChanges(await GetNotCoveredByAnyJsonAsync(), orderBy));
+        }
+
+        [Route("CoveredByAll/{orderBy?}")]
+        public async Task<IActionResult> GetCoveredByAllAsync(BreakingChangeSortFieldEnum orderBy = BreakingChangeSortFieldEnum.Id)
+        {
+            return PartialView("BreakingChanges", SortBreakingChanges(await GetCoveredByAllJsonAsync(), orderBy));
+        }
+
+        [Route("RollUp")]
+        public async Task<IActionResult> GetRollUpAsync()
+        {
+            return PartialView("RollUp", await GetRollUpJsonAsync());
+        }
+
+        /// <summary>
+        /// Returns a list of available data items to choose from
+        /// </summary>
+        private IEnumerable<SelectListItem> GetListOfDataChoices()
+        {
+            return new List<SelectListItem>
+            {
+                new SelectListItem() { Text = "Analyzers", Value="Analyzers" },
+                new SelectListItem() { Text = "Breaking changes", Value="BreakingChanges" },
+                new SelectListItem() { Text = "ApiPort summary", Value="ApiPortSummary" },
+                new SelectListItem() { Text = "Analyzers summary", Value="AnalyzersSummary" },
+                new SelectListItem() { Text = "Roll-Up", Value="RollUp", Selected = true },
+                new SelectListItem() { Text = "Not covered by any", Value="NotCoveredByAny" },
+                new SelectListItem() { Text = "Covered by all", Value = "CoveredByAll" }
+           }
+           .OrderBy(o => o.Text);
+        }
+        #endregion
+
+        #region JsonRoutes
+        [Route("ApiPortSummaryJson/{orderBy?}")]
```

Comment:
Why ApiPortSummaryJson? The fact that it is JSON is an implementation detail - the client to request it as xml and they'll get that as well.